trimmer_decorator.rb

```diff
@@ -0,0 +1,7 @@
+require './decorator'
+
+class TrimmerDecorator < Decorator
+  def correct_name
+    @nameable.correct_name[1...10] if @nameable.correct_name.length > 10
+  end
```

Comment:
- [ ] Kindly fix the range of the `correct_name` to be `[0...10]` instead of `[1...10]` as now the first character of the name will be ignored.
- [ ] Kindly add an else case for this method as if the name is less than 10 characters the method will return `nil`.

Reply:
done, as advised